pkg/scheduler/framework/plugins/noderesources/resource_allocation.go

```diff
@@ -155,3 +173,152 @@ func (r *resourceAllocationScorer) isBestEffortPod(podRequests []int64) bool {
 	}
 	return true
 }
+
+// calculateDRAExtendedResourceAllocatableRequest calculates allocatable and requested
+// quantities for extended resources backed by DRA.
+func (r *resourceAllocationScorer) calculateDRAExtendedResourceAllocatableRequest(ctx context.Context, node *v1.Node, resource v1.ResourceName, podRequest int64) (int64, int64) {
+	logger := klog.FromContext(ctx).WithValues("resource", resource, "node", node.Name)
```

Comment:
Please set values only when we need. For example:
```suggestion
	logger := klog.FromContext(ctx)
	if logger.V(5).Enabled() {
		logger = logger.WithValues("resource", resource, "node", node.Name)
	}
```

Reply:
After using logger.Error instead of `logger.Info` (see your comments below) only one `logger.V(7).Info` call is left, so I think it's not needed anymore. Let me know if you still think we need this.